ngo_design.css

```diff
@@ -0,0 +1,1315 @@
+@media (min-width: 480px) and (max-width: 800px){
```

Comment:
Where does this block end?  With the current indentation it is difficult to find this out